### PR TITLE
ASQ: updating account mapping to the new API

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to https://github.com/Particular/PlatformDevelopment/issues/723

More info: https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/118

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/docs-maintainers 
Ping @Particular/azure-storage-queues-maintainers

File: Snippets/ASQ/ASQ_7/Usage.cs [hunk 1]
```diff
@@ -33,12 +33,27 @@ void AccountNamesInsteadOfConnectionStrings(EndpointConfiguration endpointConfig
...
 
+    void MultipleAccountNamesInsteadOfConnectionStrings(EndpointConfiguration endpointConfiguration)
+    {
+        #region AzureStorageQueueUseMultipleAccountNamesInsteadOfConnectionStrings
+
+        var transport = endpointConfiguration.UseTransport<AzureStorageQueueTransport>();
+        transport
+            .UseAccountNamesInsteadOfConnectionStrings(mapping =>
+            {
+                mapping.MapLocalAccount("default"); // the account used by this endpoint
```
Comment (dvdstelt) [inline Snippets/ASQ/ASQ_7/Usage.cs]: Where does this magical string `default` come from?

Comment (Scooletz) [inline Snippets/ASQ/ASQ_7/Usage.cs]: I'll add another snippet for this and show configuration for both accounts.

File: Snippets/ASQ/ASQ_7/app.config [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (weralabaj) [inline Snippets/ASQ/ASQ_7/app.config]: do we need that?

**Comment by weralabaj** [comment]
A bit confused - should that be "account names" or "connection string names"? Aren't we mixing two things in the doco?

File: Snippets/ASQ/ASQ_7/app.config (hunk 2 above)
Comment (Scooletz) [inline Snippets/ASQ/ASQ_7/app.config]: Of course not. Removing it.

**Comment by Scooletz** [comment]
@weralabaj an `account` has a `connection string`. It's one-to-one relation. I'll rephrase this sentence.

**Pull request merged**